xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -2042,34 +2042,33 @@
   },
   "PVR.AddTimer": {
     "type": "method",
-    "description": "Adds a timer",
+    "description": "Adds a timer to record the given show one times or a timer rule to record all showings of the given show",
     "transport": "Response",
     "permission": "ControlPVR",
     "params": [
-      { "name": "broadcastid", "$ref": "Library.Id", "required": true, "description": "the broadcast id of item to record" },
-      { "name": "repeating", "type": "boolean", "default": false, "description": "the add should be done for all occurances" }
+      { "name": "broadcastid", "$ref": "Library.Id", "required": true, "description": "the broadcast id of the item to record" },
+      { "name": "timerrule", "type": "boolean", "default": false, "description": "controls whether to create a timer rule or a onetime timer" }
     ],
     "returns":  "string"
   },
   "PVR.DeleteTimer": {
     "type": "method",
-    "description": "Removes a timer",
+    "description": "Deletes a onetime timer or a timer rule",
     "transport": "Response",
     "permission": "ControlPVR",
     "params": [
-      { "name": "timerid", "$ref": "Library.Id", "required": true, "description": "the id of the timer to delete" },
-      { "name": "repeating", "type": "boolean", "default": false, "description": "the delete should be done for all occurances" }
+      { "name": "timerid", "$ref": "Library.Id", "required": true, "description": "the id of the onetime timer or timer rule to delete" }
     ],
     "returns":  "string"
   },
   "PVR.ToggleTimer": {
     "type": "method",
-    "description": "Toggles a timer",
+    "description": "Creates or deletes a onetime timer or timer rule for a given show. If it exists, it will be deletedif not, it will be created",
```

Comment:
Missing a comma too, should be `If it exists, it will be deleted, if not, it will be created`